policy_agent/policy_agent.py

```diff
@@ -0,0 +1,655 @@
+#!/usr/bin/python
+import time
+import json
+import Queue
+import logging
+from threading import Thread
+
+import requests
+
+from common.constants import *
+from pycalico.datastore_datatypes import Rules, Rule
+from pycalico.datastore_errors import ProfileNotInEndpoint, ProfileAlreadyInEndpoint
+from pycalico.datastore import DatastoreClient, PROFILE_PATH
+
+POLICY_LOG_DIR = "/var/log/calico/policy"
```

Comment:
I think this log should be under kubernetes, rather than "policy".

This agent is Kubernetes specific.